docs/using/administering/upgrading.rst

```diff
@@ -0,0 +1,40 @@
+.. _upgrading-flocker:
+
+=========
+Upgrading
+=========
+
+A Flocker cluster can be upgraded to a newer version of Flocker while preserving all data and configuration of the Flocker cluster provided certain steps are followed.
+The correct steps to follow may vary depending on the versions of Flocker being upgraded from and to.
+A common requirement for all currently supported upgrade paths is that all nodes and the control service must be running the same version of Flocker.
+
+Flocker 1.0.1
+-------------
+
+Recommended Steps
+^^^^^^^^^^^^^^^^^
+
+  #. Stop the control service and the agent services on all nodes.
+  #. Install Flocker 1.0.1 on all hosts in the Flocker cluster.
```

Comment:
we've been using nodes in place of hosts/servers, to be consistent with the code samples